### PR TITLE
[DOCUMENTAION][OTHER LANGUAGES] Update documentation, create script for Fedora

### LINKED ISSUE #30: Make .sh build script for Fedora / CentOS (Stream) / RHEL
Hi,

I'm currently running Centstream 9 and I'm having problems when building ringOS from source because it only support building on Debian or Debian-based distribution. The shell script for building ringOS doesn't support Fedora-based distribution including Centstream, RHEL, Alma, Rocky. Can you make a specified shell script for them?

Suggestion: you can make a script called configure-debian(.sh) and configure-rhel(.sh) and the configure(.sh) script to detect if it is Debian or Fedora/RHEL based distribution. Those script throw information about the current operating system to know how to build it. The build.sh or Makefile will choose the way to build based on the info about operating system. configure scripts are optionally use to install required packages. 

Hope my issue will be fixed soon.

### PR DESCRIPTION
**Pull request by hahahahacker2009**

### CODE REVIEW

**Review by hahahahacker2009 — COMMENTED** [review]
Change compile script, create script to download and compile ringOS stable & beta for Red Hat

File: Documentation.md [hunk 1]
```diff
@@ -7,35 +7,54 @@ This documentation is intended to guide you in developing, compiling, debugging
...
 Unzip the downloaded Zip folder and navigate to the contents in your Terminal (Linux) or using WSL/WSL2 on Windows ('cd')
 
+Support for Red Hat(-based) Linux distribution just come, if you want to improve consider become a contributor.
+
 --------------------------------------------------------------------------------------------------------------------------------
 ### Compile beta version from shell script
 #### How to run shell script for Debian based Linux distros:
 `./ringOS-beta.sh`
 
-If that don't work, type: `chmod +x ringOS-beta.sh` and try again.
```
Comment (Andrej123456789) [inline Documentation.md:23]: Why this is removed?

Comment (Andrej123456789) [reply Documentation.md:23]: Add `chmod +x ./ringOS-beta.sh`

Comment (hahahahacker2009) [reply Documentation.md:23]: People will know to change the exec bits before running an executable shell script.

Comment (Andrej123456789) [reply Documentation.md:23]: Are you sure?

**Comment by Andrej123456789** [comment]
Also I will upload shell script for Arch during the day and maybe this pull request will be merged soon

File: Documentation.md (hunk 1 above)
Comment (hahahahacker2009) [reply Documentation.md:23]: Yes, it is the most basic Linux computer stuff that anyone use it should know

**Comment by Andrej123456789** [comment]
@Someone8859 Here is scripts for Arch based distros. Last thing is to send message that you are ready and RedHat scripts are good so can I merge this pull request

**Comment by Andrej123456789** [comment]
This seems good, I will merge it

**Pull request merged**